packages/core/lib/commands/console/meta.js

```diff
@@ -2,10 +2,29 @@ module.exports = {
   command: "console",
   description:
     "Run a console with contract abstractions and commands available",
-  builder: {},
+  builder: {
+    url: {
+      describe: "Use specified URL for provider",
+      type: "string"
+    }
+  },
   help: {
-    usage: "truffle console [--verbose-rpc] [--require|-r <file>]",
+    usage:
+      "truffle console [--verbose-rpc] \n" +
```

Comment:
is it necessary to write 4 types of usage here? really only the `--network` and `--url` flags are incompatible.

we should be able to do something like
```
truffle console [--verbose-rpc] [--require|-r <file>] [--network <network>|--url <provider_url>]
```

no?

Reply:
I still don't understand why it needs to be 4 lines. I'm not sure it does make it clearer. Do you think it makes it clearer? Why?